applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/boards/nrf54l15pdk_nrf54l15_cpuapp_0_3_0_fixup.dtsi

```diff
@@ -0,0 +1,44 @@
+/* Ths file content is copied from the "nrf54l15pdk_nrf54l15_cpuapp_0_3_0.overlay" file defined in
+ * nrf54l15pdk_nrf54l15 board directory. The file from the board directory is not automatically used
+ * e.g. for mcuboot_secondary_app child image. Application applies content of the file manually to
+ * workaround the problem.
+ */
+
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Hi @carlescufi,
The file is a copy-paste from a Zephyr's board definition DTS overlay (see comment above for details).
Could you advise on how should we handle the license, please? I initially wanted to keep it as it's done currently (I copied file content with license and added a reference to original source), but our license check seems not to accept `Apache-2.0`.

Reply:
We decided to replace the Apache license with the Nordic 5 Clause standard license after discussion with @carlescufi